### DISCUSSION TITLE
Scraping cuepoints SLR

### DISCUSSION THREAD

**Discussion by axcpv1** [xbapps/xbvr#1203] [Q&A]
I have several SLR videos but I don't see any cue points scraped. Images and info tags are scraped fine. Just not the cuepoints... Can someone please help me. Thank you

**Comment by theRealKLH** [comment]
Cuepoints aren't scrapable from SLR. There's no scraping of cuepoints in general.

**Comment by axcpv1** [comment]
OK thanks, I guess cue points aren't scrapable then... Would be nice to have the option to grab them but maybe they're only transmitted to DeoVR and encrypted, so even with subscription it's not possible to get cuepoint export.

**Comment by Charlotte-br560** [comment]
Have you checked if cue points are present in the HTML source? Consider using [Crawlbase](https://crawlbase.com/) for easier scraping. Hope this helps!

**Comment by wheresmytissue** [comment]
Many scene cuepoints are available from https://timestamp.trade/

**Comment by ChrisGedling** [comment]
Some of them are. Try - https://www.sexlikereal.com/scenes/treasure-hunt-forest-nymph-normal-passthrough-43895 I've got 25 cue points. 

Some json data can be retrieved from https://api.sexlikereal.com/virtualreality/video/id/43895

Obviously it's possible, I don't know why it only works for a few files.
